techniques/general/G218.html

```diff
@@ -138,23 +139,23 @@ <h2>Related Resources</h2>
 
 
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
-				<p>For websites which allow users to login by emailing a link to the email address associated with the account:</p>
+				<h3>手順</h3>
+				<p>アカウントに関連付けられた電子メールアドレスにリンクをメールで送信することで利用者がログインできるウェブサイトについて:</p>
 				<ol>
-					<li>Enter a valid email address (with an account on the website) and use the email-link feature.</li>
-					<li>Check that the email is received.</li>
-					<li>Check that selecting the link opens the website.</li>
-					<li>Check that the user account is logged in.</li>
-					<li>Check that no object recognition test is used as part of the authentication process.</li>
+					<li>有効な電子メールアドレス (ウェブサイトにアカウントがあるもの) を入力し、電子メールリンク機能を使用する。</li>
+					<li>電子メールが受信されることをチェックする。</li>
+					<li>リンクを選択するとウェブサイトが開くことをチェックする。</li>
+					<li>ユーザアカウントがログインしていることをチェックする。</li>
+					<li>認証プロセスでオブジェクト認識テストが使用されていないことをチェックする。</li>
 				</ol>
 			</section>
 			<section class="test-results" id="expected-results">
-				<h3>Expected Results</h3>
+				<h3>期待される結果</h3>
 				<ul>
-					<li>#2, #3 and #4 are true.</li>
-					<li>For the AAA Accessible Authentication (Enhanced), #5 is also true.</li>
+					<li>2、3、及びが真である。</li>
+					<li>レベル AAA アクセシブルな認証 (高度) の場合、5も真である。</li>
```

Comment:
「レベル AAA アクセシブルな認証 (高度)」と一続きになっているのが、若干読みにくい感じがしました。以下、いかがでしょうか？

```suggestion
					<li>レベル AAA の、アクセシブルな認証 (高度) の場合、5 も真である。</li>
```